client/modules/accounts/templates/tax-settings/taxSettings.js

```diff
@@ -0,0 +1,75 @@
+import _ from "lodash";
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Accounts } from "/lib/collections";
+import { Accounts as AccountsSchema } from "/lib/collections/schemas/accounts";
+import { TaxEntityCodes } from "/client/collections";
+
+Template.taxSettingsPanel.helpers({
+  account() {
+    const sub = Meteor.subscribe("Accounts.single", this.member.userId);
+    if (sub.ready()) {
+      return Accounts.findOne({ _id: this.member.userId });
+    }
+    return null;
+  },
+  makeUniqueId() {
+    return `tax-settings-form-${this.member.userId}`;
+  },
+  accountsSchema() {
+    return AccountsSchema;
+  },
+  entityCodes() {
+    return _.concat(TaxEntityCodes.find().map((entityCode) => {
+      return _.assign({}, entityCode, {
```

Comment:
No specific advantage IMO, just preference. For consistency, I'll use the agreed pattern (now that I know). So, I'll change to native.